packages/cli/api-importers/openapi/openapi-ir-parser/src/openapi/v3/converters/operation/convertStreamingOperation.ts

```diff
@@ -222,7 +222,6 @@ export function isStreamingExample(
         unknown: (unknownExample) => {
             const maybeFernExample = RawSchemas.serialization.ExampleEndpointCallSchema.parse(unknownExample);
             if (!maybeFernExample.ok) {
-                context.logger.error("Failed to parse example", maybeFernExample.errors.toString());
```

Comment:
Should this log actually be removed? Was this resulting in double logs with your updated change below?

Reply:
Yup intentional -- should be removed and replaced with the log line elsewhere for a better UX